docs/14_0_0/components/datatable.md

```diff
@@ -91,7 +91,7 @@ DataTable displays data in tabular format.
 | scrollable                | false              | Boolean          | Makes data scrollable with fixed header.
 | selectAllFilteredOnly     | false              | Boolean          | When enabled, toggle select will only apply on filtered items. Default is false.
 | selection                 | null               | Object           | Reference to the selection data.
-| selectionRowMode          | new                | String           | Defines row selection mode. Valid values are "new", "add" and "none".
+| selectionRowMode          | new                | String           | Defines row select mode for checkbox based selection, valid values are 'new' (default, sets the line as the only selected line), 'add' (toggles to selection) and 'none' (the selection is untouched).
```

Comment:
I don't remember well how this option works... what do you mean by it 'toggles' the selection? If that's what it is, why did we call it 'add' ?

Reply:
That's what I see from the js doc:

```
* @typedef {"new" | "add" | "none"} PrimeFaces.widget.DataTable.SelectionRowMode Indicates how rows of a DataTable
 * may be selected, when clicking on the row itself (not the checkbox / radiobutton from p:column).
 * `new` always unselects other rows, `add` preserves the currently selected rows, and `none` disables row selection.
```